core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -618,6 +618,7 @@ write_files:
       ETCD_ADVERTISE_CLIENT_URLS=https://$advertised_hostname:2379
       ETCD_LISTEN_PEER_URLS=https://$private_ip:2380
       ETCD_INITIAL_ADVERTISE_PEER_URLS=https://$advertised_hostname:2380" >> /var/run/coreos/etcd-environment
+      ETCD_VERSION={{.Etcd.Version}}
```

Comment:
Thanks for spotting it guys. Actually first commit was both wrong and redundant, i removed it.
Cloud formation based propagation is enough to fix the problem